packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -74,6 +74,18 @@ endpoint_tests:
     type:
       - master
   - tests:
+      # This test checks whether AR's `/service` endpoint is able to route to
+      # the correct service even though the request url contains duplicate
+      # slashes.  Duplicate slashes is AFAIK not a valid RFC behaviour, but
```

Comment:
I guess you're talking about:

>  The path may consist of a sequence of path segments separated by a
   single slash "/" character.

https://www.ietf.org/rfc/rfc2396.txt